galsim/kolmogorov.py

```diff
@@ -141,7 +141,14 @@ class Kolmogorov(GSObject):
     # We convert this into T(k) = exp(-(k/k0)^5/3) for efficiency,
     # which implies 1/2 6.8839 (lambda/r0 / 2Pi)^5/3 = (1/k0)^5/3
     # k0 * lambda/r0 = 2Pi * (6.8839 / 2)^-3/5 = 2.992934
-    _k0_factor = 2.992934
+    #
+    # Update: It turns out that 6.8839/2 is actually (24 Gamma(6/5) / 5)^(5/6)
```

Comment:
Or 0.5*SBVonKarman::vkStructureFunction::magic4 :)

Reply:
Interesting.  Makes sense that this would show up there too.  I added a comment to this effect in SBVonKarman where you define magic4.